tf2_eigen/package.xml

```diff
@@ -1,20 +1,22 @@
 <?xml version="1.0"?>
-<package>
+<package format="2">
   <name>tf2_eigen</name>
   <version>0.5.12</version>
   <description>tf2_eigen</description>
   <author>Koji Terada</author>
   <maintainer email="terakoji@gmail.com">Koji Terada</maintainer>
   <license>BSD</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
   <build_depend>eigen</build_depend>
   <build_depend>geometry_msgs</build_depend>
   <build_depend>tf2</build_depend>
   <build_depend>cmake_modules</build_depend>
 
-  <run_depend>geometry_msgs</run_depend>
-  <run_depend>tf2</run_depend>
-  <run_depend>cmake_modules</run_depend>
+  <exec_depend>geometry_msgs</exec_depend>
+  <exec_depend>tf2</exec_depend>
 
```

Comment:
To be sure we agree on the meaning, here is what I interpret from this:
- Changing the package format to format 2 (being required for ament and being supported by catkin/ROS1) should be submitted upstream and not on this fork.
- Style changes should either be avoided or submitted upstream directly to avoid divergence.